analysis/parameter_tuning.py

```diff
@@ -153,6 +179,35 @@ def _find_candidates(histogram: histograms.Histogram) -> List:
         max_contributions_per_partition=linf_bounds)
 
 
+def _find_candidates_quantiles(histogram: histograms.Histogram,
+                               n_max: int) -> List[int]:
+    """Implementation of ParametersSearchStrategy.QUANTILES strategy."""
+    quantiles_to_use = [0.9, 0.95, 0.98, 0.99, 0.995]
+    candidates = histogram.quantiles(quantiles_to_use)
+    candidates.append(histogram.max_value)
+    candidates = list(set(candidates))  # remove duplicates
+    candidates.sort()
+    return candidates[:n_max]
+
+
+def _find_candidates_constant_relative_step(histogram: histograms.Histogram,
+                                            n_max: int) -> List[int]:
+    """Implementation of ParametersSearchStrategy.CONSTANT_RELATIVE_STEP
+       strategy."""
+    max_value = histogram.max_value
+    # relative step varies from 1% to 0.1%
```

Comment:
This comment is not understandable w/o context. Could you please write down in more details why candidates in that way are chosen? Something short, a few lines should be fine